Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM python:3.6-alpine
+
+RUN apk add --update --no-cache gcc g++ && pip install dumb-init
+RUN apk add --update --no-cache libxml2-dev libxslt-dev git
+
+VOLUME /var/lib/sirbot
+VOLUME /var/log/sirbot
+
+WORKDIR /app
+
+COPY . .
```

Comment:
If you want to be building this locally much, you should just copy over the `requirements.txt`, then `pip install`, then copy the rest over